docker/Dockerfile.arm64

```diff
@@ -0,0 +1,21 @@
+FROM arm64v8/ruby:2.3.7-jessie
+LABEL maintainer="Shaun Jackman <sjackman@gmail.com>"
+
+ENV PATH=/home/linuxbrew/.linuxbrew/bin:/home/linuxbrew/.linuxbrew/sbin:$PATH \
+    HOMEBREW_BUILD_BOTTLE=1 \
+    HOMEBREW_DEVELOPER=1 \
+    HOMEBREW_NO_ANALYTICS=1 \
+    HOMEBREW_NO_AUTO_UPDATE=1 \
+    HOMEBREW_RUBY_PATH=/usr/local/bin/ruby
+
+RUN sed -i '/debian-security/d' /etc/apt/sources.list \
+    && apt-get update \
+    && apt-get install -y --no-install-recommends \
+        ca-certificates curl file g++ git-core locales make patch \
+    && rm -rf /var/lib/apt/lists/* \
+    && localedef -i en_US -f UTF-8 en_US.UTF-8 \
+    && useradd -m -s /bin/bash linuxbrew \
+    && git clone --depth=1 https://github.com/Homebrew/brew /home/linuxbrew/.linuxbrew/Homebrew \
+    && mkdir /home/linuxbrew/.linuxbrew/bin \
+    && ln -s ../Homebrew/bin/brew /home/linuxbrew/.linuxbrew/bin/ \
+    && brew tap homebrew/portable-ruby
```

Comment:
Don't you need to change users to linuxbrew? Won't all the `/home/linuxbrew` dirs be owned by root?

Also, I think you need to tap `homebrew/core` too. Also, the other docker files run `brew analytics off` explicitly. And install patchelf, etc.

Reply:
> Don't you need to change users to linuxbrew? Won't all the `/home/linuxbrew` dirs be owned by root?

We only care about what ends up in the bottle tarball. The files in the tarball will owned by `root:root` rather than `linuxbrew:linuxbrew` as we've been doing until now. So that is a change. Normally the owner of the file stored in the tarball is ignored when you extract the tarball.

Reply:
> Also, I think you need to tap `homebrew/core` too.

That's done automatically.

Reply:
> Also, the other docker files run `brew analytics off` explicitly.

We have `ENV HOMEBREW_NO_ANALYTICS=1`

Reply:
> We only care about what ends up in the bottle tarball. The files in the tarball will owned by `root:root` rather than `linuxbrew:linuxbrew` as we've been doing until now. So that is a change. Normally the owner of the file stored in the tarball is ignored when you extract the tarball.

We could always just `chown -R linuxbrew:linuxbrew` and add the linuxbrew user back... it's worth testing that un-priveledged users can build etc., IMO.

Reply:
The `linuxbrew` user is still created. It's just not the current user.

> it's worth testing that un-priveledged users can build etc., IMO.

You mean that un-privileged users can run `brew portable-package` to build portable bottles?

Reply:
Yes... but maybe that's dumb. As long as poured bottle perms get sorted out correctly, it probably doesn't matter.